src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getCatalogWidth ($cardWidth, $cardTopMargin) {
+  @return 4 * $cardWidth + 3 * $cardTopMargin;
```

Comment:
what it 4 and 3?

Reply:
the number of cards, for example, in row 4, we will put 1, there will be 1 card in the row

Reply:
let's create a variable, and avoid magic numbers)